apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -78,13 +79,31 @@ app = do
                        }
         paymentView = payments paymentProps
         paymentDetailView = paymentDetail paymentProps
-    pure $ render self logger searchView paymentView paymentDetailView isPersonating
+        creditCardUpdateInputs =
+          CreditCardUpdateView.RouteWrapperContentInputs
+            { creditCards: fromMaybe mempty $ state.activeUser <#> _.creditCards
+            , logger: logger
+            }
+        updateCreditCardRoute =
```

Comment:
Maybe we could refactor this bit to use `defaultRouteElement` or `routeElement`? Sorry I don't know if it was like now before my approval, I missed it